pkg/occlient/occlient_test.go

```diff
@@ -1952,6 +1973,227 @@ func TestNewAppS2I(t *testing.T) {
 	}
 }
 
+func TestIsTagInImageStream(t *testing.T) {
+	tests := []struct {
+		name        string
+		imagestream imagev1.ImageStream
+		imageTag    string
+		wantErr     bool
+		want        bool
+	}{
+		{
+			name:        "Case: Valid image and image tag",
+			imagestream: *fakeImageStream("foo", "openshift", []string{"latest", "3.5"}),
+			imageTag:    "3.5",
+			want:        true,
+		},
+		{
+			name:        "Case: Invalid image tag",
+			imagestream: *fakeImageStream("bar", "testing", []string{"latest"}),
+			imageTag:    "0.1",
+			want:        false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+
+			got := isTagInImageStream(tt.imagestream, tt.imageTag)
+
+			if got != tt.want {
+				t.Errorf("GetImageStream() = %#v, want %#v\n\n", got, tt)
+			}
+		})
+	}
+}
+func fakeImageStreamImage(imageName string, ports []string) *imagev1.ImageStreamImage {
```

Comment:
Move helper functions to the top above all the test functions.

Reply:
Ack